modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -172,8 +175,11 @@ The OTLP receiver ingests data using the OpenTelemetry protocol (OTLP).
           receivers: [otlp]
 ----
 <1> The OTLP gRPC endpoint. If omitted, the default `+0.0.0.0:4317+` is used.
-<2> The OTLP HTTP endpoint. If omitted, the default `+0.0.0.0:4318+` is used.
-<3> The TLS server side configuration. Defines paths to TLS certificates. If omitted, TLS is disabled.
+<2> The TLS server side configuration. Defines paths to TLS certificates. If omitted, TLS is disabled.
+<3> Path to the TLS certificate to use by the server to verify a client certificate. (optional) This sets the ClientCAs and ClientAuth to RequireAndVerifyClientCert in the TLSConfig. Please refer to link:https://godoc.org/crypto/tls#Config[Golang TLS Config] for more information.
+<4> Specifies the duration after which the certificate will be reloaded. If not set, it will never be reloaded. Accepts a duration string, valid time units are `ns`, `us` (or `µs`), `ms`, `s`, `m`, `h`.
```

Comment:
I wouldn't recommend using `duration` unless clearly specifying the  `duration of [something]`. Perhaps replace it `time interval` like in my suggestion below?
```suggestion
<4> Specifies the time interval at which the certificate is reloaded. If not set, the certificate is never reloaded. Accepts a string with valid units of time such as `ns`, `us` (or `µs`), `ms`, `s`, `m`, `h`.
```